### ISSUE TITLE
Feature to add evidence, and suspect pictures.

### ISSUE THREAD

**Issue by CheatInNeed** [ExpDev07/legacy-rp-cad#10]
Feature to add ex. 'gyazo' links to add evidence, if the suspect takes the case to court.

**Comment by ExpDev07** [comment]
Good idea! Would this fall under arrest reports (https://github.com/ExpDev07/legacy-rp-cad/issues/4) or be its own thing somewhere? I'll add the option to just add any links so that youtube, imgur, etc can also be provided.
